draft-irtf-cfrg-fiat-shamir.md

```diff
@@ -107,59 +113,128 @@ This will be expanded in future versions of this specification.
 We describe how to construct non-interactive proofs for sigma protocols.
 The Fiat-Shamir transformation is parametrized by:
 
+- a `SigmaProtocol`, which specifies an interactive 3-message protocol;
```

Comment:
Consider adding a reference to the sigma draft
```suggestion
- a `SigmaProtocol`, which specifies an interactive 3-message protocol as defined in {{Section 2 of !SIGMA}}
```

Reply:
done!